app/assets/stylesheets/components/_footer.scss

```diff
@@ -0,0 +1,76 @@
+.footer__global {
+  background-color: $c-blue;
+  color: white;
+}
+
+.footer__text,
+.footer__logo {
+  float: left;
+}
+
+.footer__text {
+  font-family: $f-body;
+  font-weight: 400;
+  line-height: 1.8;
+  min-width: 203px;
+  width: rd(203, 415);
+
+  p {
+    @include fluid-size("font-size", 10px, 18px);
+  }
+
+  small {
+    @include fluid-size("font-size", 10px, 13px);
+  }
+}
+
+.footer__logo {
+  display: block;
+  float: right;
+  height: auto;
+  margin: 0 auto;
+  max-width: 200px;
```

Comment:
the png you're using is a lot bigger than 200px wide - might want to cut down the image file so it's not bigger than it needs to be